### PR TITLE
ISPN-10995 upgrade infinispan quickstart to latest

### PR DESCRIPTION
**Pull request by karesti**
This PR fixes the Infinispan client quickstart for the latest 10.x final version

https://issues.jboss.org/browse/ISPN-10995

### CODE REVIEW

File: infinispan-client-quickstart/README.md [hunk 1]
```diff
@@ -6,15 +6,14 @@ This example showcases how to use the Infinispan client with Quarkus.
 
 You have two options:
 
-**Option 1:** Running with Docker `docker run -it -p 11222:11222 jboss/infinispan-server:latest`
+**Option 1:** Running with Docker `docker run -it -p 11222:11222 -e USER="Titus Bramble" -e PASS="Shambles" infinispan/server:latest`
```
Comment (gsmet) [inline infinispan-client-quickstart/README.md]: You have a space in the user name here and not in the Docker file. I don't think that's what you want?

File: infinispan-client-quickstart/src/main/resources/application.properties [hunk 2]
```diff
@@ -1,3 +1,5 @@
 # Your configuration properties
 quarkus.http.port=8081
-quarkus.infinispan-client.server-list=localhost:11222
\ No newline at end of file
+quarkus.infinispan-client.server-list=localhost:11222
+## Docker 4 Mac workaround
+quarkus.infinispan-client.client-intelligence=BASIC
```
Comment (gsmet) [inline infinispan-client-quickstart/src/main/resources/application.properties]: How does it authenticate now that we have a username and password?

Comment (karesti) [reply infinispan-client-quickstart/src/main/resources/application.properties]: I don't know why it was working just before without any auth configured. but I'm getting very weird stuff in my laptop.
Would you mind testing the quickstart and the docker-compose file from your side, please?

File: infinispan-client-quickstart/README.md (hunk 1 above)
Comment (karesti) [reply infinispan-client-quickstart/README.md]: I changed for the space one

File: infinispan-client-quickstart/src/main/resources/application.properties (hunk 2 above)
Comment (cescoffier) [reply infinispan-client-quickstart/src/main/resources/application.properties]: on it.

Comment (cescoffier) [reply infinispan-client-quickstart/src/main/resources/application.properties]: works!

**Pull request merged**